### ISSUE TITLE
change gitea username

### ISSUE THREAD

**Issue by nimakaviani** [cnoe-io/idpbuilder#304]
Currently, the username for gitea is `giteaAdmin`. This causes some complications when using it for logging in and utilizing images in the gitea OCI registry. See here: https://github.com/cnoe-io/idpbuilder/pull/302#discussion_r1640460167

To simplify things, we should just rename the username from `giteaAdmin` to `root` and update references and instructions where necessary.

When fixing this, make sure you search for both `giteaAdmin` and `giteaadmin` references to fix them.

**Comment by blakeromano** [comment]
```sh
k logs my-gitea-8674989bbc-xp4cq -c configure-gitea
==== BEGIN GITEA CONFIGURATION ====
2024/06/18 03:49:49 ...2@v2.27.1/command.go:272:Run() [I] PING DATABASE sqlite3
2024/06/18 03:49:49 cmd/migrate.go:33:runMigrate() [I] AppPath: /usr/local/bin/gitea
2024/06/18 03:49:49 cmd/migrate.go:34:runMigrate() [I] AppWorkPath: /data
2024/06/18 03:49:49 cmd/migrate.go:35:runMigrate() [I] Custom path: /data/gitea
2024/06/18 03:49:49 cmd/migrate.go:36:runMigrate() [I] Log path: /data/log
2024/06/18 03:49:49 cmd/migrate.go:37:runMigrate() [I] Configuration file: /data/gitea/conf/app.ini
2024/06/18 03:49:49 ...2@v2.27.1/command.go:272:Run() [W] Table system_setting Column version db default is , struct default is 1
No admin user 'admin' found. Creating now...
Command error: CreateUser: name is reserved [name: admin]
```

I am not sure how feasible this is with gitea. This was based off a quick look at trying an `idpbuilder create` after changing the admin username

**Comment by nimakaviani** [comment]
interesting! yeah. looks like the username cannot come from [this reserved list](https://github.com/go-gitea/gitea/blob/21783a5752f518e579d8fe48b33504a16674ee17/models/user/user.go#L521-L559). maybe we call it `administrator` or `root`? root might be shorter and easier. kind of a bummer though. would have been nice to have the same username between argocd and gitea. thoughts on calling it `root`?

**Comment by tapas4java** [comment]
Can we move keycloak as core application and let ArgoCD and Gitea use managed accounts similar to other apps? I think when we add deployment to remote cluster support, this workflow will be seemless.

**Comment by nimakaviani** [comment]
yeah we discussed this in the past. Problem is, we have users who have done integrations with other SSO providers, including Okta, Cognito, etc. Bringing Keycloak into the mix as a core package, makes it confusing if folks choose to enable alternative SSO solutions in other stacks.

**Comment by nabuskey** [comment]
`root` sounds better to me. Shorter and easier to type :)

**Comment by cmoulliard** [comment]
> `root` sounds better to me. Shorter and easier to type :)

`admin` is the default standard ;-)

**Comment by nimakaviani** [comment]
@cmoulliard yeah `admin` would have been great but it is a reserved word. see [this](https://github.com/cnoe-io/idpbuilder/issues/304#issuecomment-2175223164)

**Comment by cmoulliard** [comment]
Let's then use one of the followings: `adminuser`, `gitadmin`, `supervisor` ?

**Comment by abangser** [comment]
we use `gitea_admin` if that helps at all.
